pykrige/uk.py

```diff
@@ -1,1076 +1,1109 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-from __future__ import unicode_literals

```

Comment:
Same issue with the diff. Maybe it's due to the change in permissions?

Reply:
Rebasing resolved this. Diffs are now clearer.